### ISSUE TITLE
Add focus indicator on vf-tabs to assist keyboard navigation

### ISSUE THREAD

**Issue by nikiforosk** [visual-framework/vf-core#1759]
The [vf-tabs component](https://stable.visual-framework.dev/components/vf-tabs/) does not have visual a focus indicator to help people who navigate with the keyboard.

As mentioned in [the keyboard testing guidelines of Webaim](https://webaim.org/techniques/keyboard/#problems):

> A keyboard user typically uses the Tab key to navigate through interactive elements on a web page—links, buttons, fields for inputting text, etc. When an item is tabbed to, it has keyboard "focus" and can be activated or manipulated with the keyboard. A sighted keyboard user must be provided with a visual indicator of the element that currently has keyboard focus. Focus indicators are provided automatically by web browsers. While their appearance varies based on the browser, the focus indicator is typically shown as a border or highlight (called an outline) around the focused element.

[The vf-navigation component](https://stable.visual-framework.dev/components/vf-navigation/) shows an outline for the active tab:
![vf-navigation-outline-active-tab-Screenshot 2022-02-14 at 11 24 06](https://user-images.githubusercontent.com/20280139/153856762-6d9602d4-3d2a-47de-90da-70e86d493e7f.png)

However, when the keyboard focus moves to the active tab of vf-tabs (labelled "Section" below) the outline isn't shown:
![vf-tab-no-outline-active-tab-Screenshot 2022-02-14 at 11 24 20](https://user-images.githubusercontent.com/20280139/153856866-b08b32dc-278b-4eb7-a6eb-49926ad31b7a.png)

We should add the outline to vf-tabs too to assist keyboard navigation.

We came across this issue while we were testing keyboard navigation on the front page of the EBI training online tutorial: https://www.ebi.ac.uk/training/online/courses/bioinformatics-terrified/

**Comment by nikiforosk** [comment]
This is a duplicate of https://github.com/visual-framework/vf-core/issues/1758 so I'm closing it.
